python/semantic_kernel/connectors/memory/chroma/chroma_memory_store.py

```diff
@@ -100,11 +100,14 @@ async def get_collection_async(
         self, collection_name: str
     ) -> Optional["Collection"]:
         try:
-            # Current version of ChromeDB rejects camel case collection names.
-            return self._client.get_collection(
-                name=camel_to_snake(collection_name),
-                embedding_function=self._default_embedding_function,
-            )
+            if camel_to_snake(collection_name) in await self.get_collections_async():
```

Comment:
Two further thoughts:
- it may be helpful to add a comment explaining why the new check is necessary, especially since it addresses a very specific issue related to using a remote instance of ChromaDB.
- is there any test coverage that needs to be updated?